lib/nacs-seq/manager.h

```diff
@@ -139,6 +139,8 @@ class Manager {
         };
         std::unique_ptr<Dump> dump;
 
+        uint32_t refresh_device_restart(const char *dname);
```

Comment:
This is ok for now. But just like most of the current APIs for running the sequence, it needs to be changed to an asynchronous interface instead.